docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3359,6 +3363,21 @@ for the table entry "executes" the direct resource appropriately.
         * if **set**: The response includes the counter entry's value read from
           the target.
 
+* `meter_counter_data` field
+    * `WriteRequest (INSERT)`
+        * if **unset**: The initial value for all 3 counter entries is the default
+          (0).
+        * if **set**: The initial value for all 3 counter entries is the default (0).
+        Sub-field values, if any set, are ignored. 
```

Comment:
nit: inconsistent indentation with the rest

Reply:
I believed indentation will be taken care of by the Madoko tool.